alibi/explainers/cfrl_tabular.py

```diff
@@ -462,7 +465,7 @@ def _diversity(self,
             X_cf, Y_m_cf, Y_t = results["X_cf"], results["Y_m_cf"], results["Y_t"]
 
             # Select only counterfactuals where prediction matches the target.
-            X_cf = X_cf[Y_t == Y_m_cf]
+            X_cf = X_cf[Y_t == Y_m_cf]  # type: ignore # TODO: fix me
```

Comment:
`alibi/explainers/cfrl_tabular.py:468: error: Value of type "Optional[Any]" is not indexable` - felt like would need more refactoring